src/e2e/java/teammates/e2e/cases/lnp/BaseLNPTestCase.java

```diff
@@ -79,6 +85,16 @@
      */
     protected abstract Map<String, String> getRequestParameters();
 
+    /**
+     * Returns the limit of the ratio of the error samples against total number of samples.
```

Comment:
`Returns the maximum allowable threshold for the number of failed requests to the test endpoint.`

Also add that the value should be a percentage (real-valued number between 0 to 100), not a ratio